notebooks/Connected Components -- FastSV.ipynb

```diff
@@ -0,0 +1,3307 @@
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td>1</td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>11</th>\n",
+       "      <td></td>\n",
+       "      <td>1</td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div></details></div>"
+      ],
+      "text/plain": [
+       "\"M_1\"          nvals  nrows  ncols  dtype     format\n",
+       "grblas.Matrix     12     12     12  INT32  csr (iso)\n",
+       "----------------------------------------------------\n",
+       "   0  1  2  3  4  5  6  7  8  9  10 11\n",
+       "0                                 1   \n",
+       "1                     1               \n",
+       "2                  1                  \n",
+       "3                              1      \n",
+       "4            1                        \n",
+       "5         1                           \n",
+       "6   1                                 \n",
+       "7                           1         \n",
+       "8                                    1\n",
+       "9                        1            \n",
+       "10              1                     \n",
+       "11     1                              "
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "P"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "714f35b5-27a5-46c9-b932-d059dcee8dc7",
+   "metadata": {},
+   "source": [
+    "Now let us transform the adjacency matrix $\\mathbf{A}$, using the permutation matrix $\\mathbf{P}$, into \n",
+    "$$\\mathbf{A}' = \\mathbf{P}^{\\mathrm{T}} \\cdot \\mathbf{A} \\cdot \\mathbf{P}.$$\n",
+    "This ensures that the graph edges are preserved, that is,\n",
+    "\\begin{equation}\n",
+    "\\boxed{A_{i\\,j} = A'_{p(i)\\,p(j)}}\n",
+    "\\end{equation}\n",
+    "for all $i$, $j$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "638d09e6-d33d-4f25-8144-0dba8e3f7988",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-12-04T09:37:08.234350Z",
+     "iopub.status.busy": "2021-12-04T09:37:08.233351Z",
+     "iopub.status.idle": "2021-12-04T09:37:08.246348Z",
+     "shell.execute_reply": "2021-12-04T09:37:08.244347Z",
+     "shell.execute_reply.started": "2021-12-04T09:37:08.234350Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "AA = A.dup()\n",
+    "AA << P.T @ A @ P"
```

Comment:
@eriknw 
This notebook now contains an example of a compound expression using the `matmul` operator.